docker-compose.yml

```
@@ -0,0 +1,43 @@
+version: '3.8'
+services:
+  hashtopolis:
+    container_name: hashtopolis
+    build: .
+    restart: always
+    volumes:
+      - ./src/import/:/var/www/html/import/:Z
+      - ./src/files/:/var/www/html/files/:Z
+      - ./data/conf.php:/var/www/html/inc/conf.php:Z
+    depends_on:
+      - db
+    ports:
+      - 8080:80
+    cap_drop:
+      - ALL
+    cap_add:
+      # used to change files since non-priv user
+      - CHOWN
+      # used by apache to drop privs
+      - CAP_SETGID
+      - CAP_SETUID
+  db:
+    container_name: db
+    image: mysql:5.7
+    restart: always
+    volumes:
+      - ./data/db:/var/lib/mysql:Z
```

Change this to persistent volume, because the mysql container does some change owning and permission checking on the files it is better to not have it on the host os. For example having the files on a CIFS share will cause errors.

```suggestion
      - db:/var/lib/mysql
```